mltrace/cli/cli.py

```diff
@@ -274,16 +294,27 @@ def flag(output_id: str, address: str = ""):
 
 
 @mltrace.command("unflag")
-@click.argument("output_id")
+@click.option("--output_id", help="Output ID to unflag")
+@click.option("--all", is_flag=True)
 @click.option("--address", help="Database server address")
-def unflag(output_id: str, address: str = ""):
+def unflag(output_id: str = "", all: bool = False, address: str = ""):
```

Comment:
What if they set both `all` and an `output_id`? Check for this in the beginning and raise an error.

Reply:
done.